.github/workflows/atd_moped_build_stage.yml

```diff
@@ -15,16 +15,17 @@ jobs:
     name: Build Editor
     runs-on: ubuntu-latest
     steps:
+      # Get the code first
+      - name: "Checkout"
+        uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
           python-version: "3.8"
           architecture: "x64"
-      - uses: actions/setup-node@v2
+      # Install node according to our .nvmrc file
+      - uses: actions/setup-node@v3
```

Comment:
bumping from `setup-node@v2` to `setup-node@v3`, which merely dropped support for pre-v16 versions of node.